src/app/app.module.ts

```diff
@@ -21,9 +21,10 @@ import { NgxsModule } from '@ngxs/store';
 import { SyncModule } from './sync/sync.module';
 import { NgHttpLoaderModule } from 'ng-http-loader';
 import { TransactionModule } from './transaction/transaction.module';
+import { AboutComponent } from './about/about.component';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/personalmoney/app-supabase/pullRequest?prid=7587366)
```suggestion
import { AboutComponent } from "./about/about.component";
```